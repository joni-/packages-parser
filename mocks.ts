import { Paragraph } from "./parser/parser";

export const paragraphs: Paragraph[] = [
  {
    name: "libws-commons-util-java",
    description: {
      synopsis: "Common utilities from the Apache Web Services Project",
      description: `This is a small collection of utility classes, that allow high
performance XML processing based on SAX.`,
    },
  },
  {
    name: "python-pkg-resources",
    description: {
      synopsis: "Package Discovery and Resource Access using pkg_resources",
      description: `The pkg_resources module provides an API for Python libraries to
access their resource files, and for extensible applications and
frameworks to automatically discover plugins.  It also provides
runtime support for using C extensions that are inside zipfile-format
eggs, support for merging packages that have separately-distributed
modules or subpackages, and APIs for managing Python's current
"working set" of active packages.`,
    },
  },
];
