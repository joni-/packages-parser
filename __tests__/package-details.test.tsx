import React from "react";
import renderer from "react-test-renderer";
import { paragraph1 } from "../mocks";
import PackageDetails from "../pages/[pkg]";

describe("PackageDetails", () => {
  it("renders page without changes", () => {
    const tree = renderer
      .create(<PackageDetails paragraph={paragraph1} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
