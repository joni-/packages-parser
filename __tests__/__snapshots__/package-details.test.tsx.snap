// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PackageDetails renders page without changes 1`] = `
<div>
  <h1>
    libws-commons-util-java
  </h1>
  <h3>
    Common utilities from the Apache Web Services Project
  </h3>
  <p>
    This is a small collection of utility classes, that allow high
performance XML processing based on SAX.
  </p>
</div>
`;
